### PR TITLE
Change contributor status

### PR DESCRIPTION
**Pull request by pierlo-upitup**
cc @aldipower

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

File: Authors.md [hunk 1]
```diff
@@ -12,5 +12,5 @@ Orlando Hohmeier | orlando@mesosphere.com | Current Maintainer
 Pierluigi Cau | pierlo@mesosphere.com | Current Maintainer
 Philipp Hinrichsen | philipph@mesosphere.com | Current Maintainer
 Philip Norman | philip@mesosphere.com | Current Maintainer
-Felix Gertz | felix@mesosphere.com | Current Maintainer
+Felix Gertz | https://github.com/aldipower | Contributor
```
Comment (orlandohohmeier) [inline Authors.md]: I wonder if we should leave the email column empty... What do you think?

Comment (pierlo-upitup) [inline Authors.md]: I really don't feel like adding personal emails either. I think the github profile makes sense as it gives a way to get in touch with the contributor.

**Comment by philipnrmn** [comment]
😢

**Pull request merged**